configuration.yaml

```diff
@@ -1,24 +1,11 @@
 # Name of your Slack (Deprecated in favour of the SLACK_NAME env var)
 slack_name: rands-leadership
 
-# Name of environment variable holding a valid Slackbot API token
-slackbot_api_token_env_varname: SB_TOKEN
-
-# Name of environment variable holding a valid general-purpose API token
-api_token_env_varname: API_TOKEN
-
 # Name of environment variable that determines if destalinator should
 # actually do things to a live slack. Without this environment variable,
 # everything will output in a "dry run" fashion (debug messages should take place,
 # but no changes or messages should affect a live Slack).
-destalinator_activated_env_varname: DESTALINATOR_ACTIVATED
-
-# Name of environment variable determining whether output debug mode is enabled
-output_debug_env_varname: DESTALINATOR_SLACK_VERBOSE
-
-# Name of environment variable for the earliest date to archive stale channels.
-# If this is set, it should be of the form "yyyy-mm-dd" (e.g. "2017-02-19").
-earliest_archive_date_env_varname: EARLIEST_ARCHIVE_DATE
+destalinator_activated: false
```

Comment:
Agreed, want me to throw in the `DESTALINATOR_` prefix in this PR? I couldn't tell if you talking about it internally meant you were ready to go forward with it. If so, I'll change the getter for this to be just `activated` as you said.